tests/testthat/test-againstCdm.R

```diff
@@ -109,18 +100,19 @@ test_that("Cohort diagnostics in incremental mode", {
       runTimeSeries = TRUE,
       minCellCount = minCellCountValue,
       incremental = TRUE,
-      incrementalFolder = file.path(folder, "incremental")
+      incrementalFolder = file.path(folder, "incremental"),
+      covariateSettings = covariateSettings,
+      temporalCovariateSettings = temporalCovariateSettings
     )
   )
   testthat::expect_lt(secondTime[1], firstTime[1])
-  
+
   # generate premerged file
   preMergeDiagnosticsFiles(dataFolder = file.path(folder, "export"))
   testthat::expect_true(file.exists(file.path(folder, "export", "PreMerged.RData")))
-  
+
   output <- read.csv(file.path(folder, "export", "covariate_value.csv"))
   expect_equal(output$sum_value[2], -minCellCountValue)
```

Comment:
These last 2 tests fail on the non-eunomia platforms but I'm not sure what they're actually trying to test.

Reply:
These are specific to the exact data in Eunomia and should be removed. I think we're good with the remaining tests.